pkg/controller/operands/dashboard.go

```diff
@@ -0,0 +1,152 @@
+package operands
+
+import (
+	"errors"
+	v1 "k8s.io/api/core/v1"
+	"os"
+	filepath "path/filepath"
+	"reflect"
+	"strings"
+
+	log "github.com/go-logr/logr"
+	hcov1beta1 "github.com/kubevirt/hyperconverged-cluster-operator/pkg/apis/hco/v1beta1"
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/controller/common"
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/util"
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Dashboard ConfigMaps contain json definitions of OCP UI
+const (
+	dashboardManifestLocationVarName = "DASHBOARD_FILES_LOCATION"
+	dashboardManifestLocationDefault = "./dashboard"
+)
+
+func newDashboardHandler(Client client.Client, Scheme *runtime.Scheme, required *v1.ConfigMap) Operand {
+	h := &genericOperand{
+		Client: Client,
+		Scheme: Scheme,
+		crType: "ConfigMap",
+		isCr:   false,
+		// Previous versions used to have HCO-operator (scope namespace)
+		// as the owner of NetworkAddons (scope cluster).
```

Comment:
Is this the right comment? Not sure we need a comment at all here. It's a new "component" and so there was never an owner for it.